packages/vue-ds/src/components/SplitButton/SplitButton.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <nav class="SplitButton">
+    <Button :color="color" :size="size" @click="onClickButton">
```

Comment:
I saw that you put `aria-expanded` at React component. At Vue, you can use dynamic props using that strategy: 
```JS
<Button
   v-bind="{'aria-expanded': isOpen}"></Button>
```

Reply:
You are amazing @elizeulouzeiro